cockatrice/src/carddbparser/cockatricexml3.cpp

```diff
@@ -72,24 +73,24 @@ void CockatriceXml3Parser::loadSetsFromXml(QXmlStreamReader &xml)
             break;
         }
 
-        if (xml.name() == "set") {
+        if (xml.name().toString() == "set") {
             QString shortName, longName, setType;
             QDate releaseDate;
             while (!xml.atEnd()) {
                 if (xml.readNext() == QXmlStreamReader::EndElement) {
                     break;
                 }
 
-                if (xml.name() == "name") {
+                if (xml.name().toString() == "name") {
                     shortName = xml.readElementText(QXmlStreamReader::IncludeChildElements);
-                } else if (xml.name() == "longname") {
+                } else if (xml.name().toString() == "longname") {
                     longName = xml.readElementText(QXmlStreamReader::IncludeChildElements);
-                } else if (xml.name() == "settype") {
+                } else if (xml.name().toString() == "settype") {
                     setType = xml.readElementText(QXmlStreamReader::IncludeChildElements);
-                } else if (xml.name() == "releasedate") {
+                } else if (xml.name().toString() == "releasedate") {
                     releaseDate =
                         QDate::fromString(xml.readElementText(QXmlStreamReader::IncludeChildElements), Qt::ISODate);
-                } else if (xml.name() != "") {
+                } else if (xml.name().toString() != "") {
```

Comment:
comparisons to "" are probably useless, .isEmpty() or something would make more sense